src/components/typography/BaseTypography.tsx

```diff
@@ -54,7 +54,7 @@ export const BaseTypography: React.FC<OwnProps> = props => {
     : [props.fontStyle, fontStyle];
 
   return (
-    <Text {...props} style={style}>
+    <Text allowFontScaling={false} {...props} style={style}>
```

Comment:
what about wiring this to the experimental context too?

Reply:
Note that `io-app` has a default prop with `allowFontScaling` set to false. So it's kind of a redundant prop for proper testing in the `example` app.